releasetasks/templates/fennec/version_bump.yml.tmpl

```diff
@@ -0,0 +1,45 @@
+{% set buildername = "release-{}-{}_version_bump".format(branch, product) %}
```

Comment:
The script itself you mean? Am not sure yet. The builder is set to False anyway until I add the necessary configs within mozharness and buildbot-configs.